tests/integration/goldens/credentials/setup.py

```diff
@@ -55,9 +55,7 @@
     if package.startswith("google")
 ]
 
-namespaces = ["google"]
-if "google.cloud" in packages:
-    namespaces.append("google.cloud")
+namespaces = ["google", "google.iam"]
```

Comment:
I don't quite grok everything this PR implies, but here we used to be adding `"google.cloud"` but now you're adding `"google.iam"`. Is this intentional?

Reply:
`google.iam` is the correct namespace as the source lives in [google/iam](https://github.com/googleapis/googleapis/blob/master/google/iam). Historically, we have used the `python-gapic-namespace` option to override the default namespace when the API is part of cloud
https://github.com/googleapis/googleapis/blob/master/google/iam/v2/BUILD.bazel#L181. I'll push a commit to add that option.

Reply:
OK, thanks for clarifying.